plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -70,12 +74,24 @@ fu_ccgx_hpi_device_to_string (FuDevice *device, guint idt, GString *str)
 }
 
 typedef struct {
-	guint8			 mode;
-	guint16			 addr;
-	guint8			*buf;
-	gsize			 bufsz;
+	guint8 		 mode;
+	guint16 	 addr;
+	guint8 		*buf;
+	gsize 		 bufsz;
```

Comment:
same here

Reply:
@hughsie 
`same here`

In line 77 ~ 80, I think that mode,addr,buf,bufsz are vertically lineuped.
Am I wrong?

Reply:
it's currently space tab space.

Reply:
@hughsie 
`it's currently space tab space.`

Which one is right between tab + space or space + tab?
To lineup *, I think one space should be needed.

Reply:
as a rule, space never comes before tab. It should be tab space for no pointers, and just tab for pointers.